js/main.js

```diff
@@ -0,0 +1,8 @@
+(function main() {
+	var numberOfGroups = 4;
+	var splitter       = new Splitter(numberOfGroups);
+	var displayer      = new Displayer(numberOfGroups);
+	var buttonAction   = new ButtonAction(splitter, displayer);
+	var button         = new Button("split", buttonAction);
+	button.initialize("attendees", "results");
```

Comment:
I tend to use 2 spaces, but I know some others prefer 4. It seems you used 2 spaces all throughout the source and specs except this one :police_car: :cop: